api/http/error.go

```diff
@@ -30,6 +31,10 @@ func (e *Error) Error() string {
 	}
 }
 
+func (e *Error) ToError() error {
```

Comment:
```suggestion
func (e *Error) Unwrap() error {
```
Unwrap was introduced since go 1.13

Reply:
Unwrap seems to be related to the case when this `Error` struct just wraps another `error`. This function just creates `error` from the string composed from wrapped fields.

Reply:
`ToError` func on an `Error` struct that itself implements =error= is still confusing to me. I think think that Unwrap would be better.